_includes/sections/working-here/managing-your-day-to-day-work.md

```diff
@@ -158,6 +158,39 @@ incurring them.
 We manage expenses using Xero. For more information about how to do this, see
 the [guide on claiming expenses](/guides/claiming-expenses)
 
+#### Travel and Accommodation Policy
+
+Currently the default for everyone in the organisation is working from home
+which means, strictly speaking, that no-one is commuting. For this period, we
+will consider commuting as regular travel to an office space without a business
+critical need.
+
+Commuting costs are the responsibility of individuals, while dxw will fund
+business critical travel and accommodation.
+
+A business critical need is defined as:
+
+- Client meetings
+- Meetings with your line manager (e.g. during probation, monthly check-ins,
+  quarterly reviews, post parental / sick leave)
+- Team meetups (e.g. design community, tech forum, away days, Board meetings)
+- All company in-person events
```

Comment:
Potentially ambiguous: could be construed as "all-company events" i.e. things where everyone meets up, which I think is not the intention.

Reply:
Given that the list mentions individual meetings, then team meetings, I suspect that this bullet is about events that are for everyone in dxw.

So maybe:
 - All-staff in-person events